src/components/layout/SidebarLinks.tsx

```diff
@@ -60,7 +73,10 @@ const renderLink = (link: SidebarLink, child?: boolean) => {
       )}
       defaultOpened={link.defaultOpened}
     >
-      {link.children && link.children.map((link) => renderLink(link, true))}
+      {link.children &&
+        link.children.map((link) =>
+          renderLink(link, { child: true, onLinkClick }),
```

Comment:
nit: I'd rename this to sound more like a boolean, `child` or `children` could be confused with a React node.

```suggestion
          renderLink(link, { isChild: true, onLinkClick }),
```

Reply:
Nice catch, fixed in aa9757619c6cdc04af1f7102e4437be59b9f5afd. Thanks!